_pytest/_code/source.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import generators
+from __future__ import absolute_import, division, generators, print_function
```

Comment:
@nicoddemus that seems to be there exactly to support generators on old python versions, i think it can be disabled - perhaps we can even remove the parameter